DigitalEdge.Repository/Enitity/Appointment.cs

```diff
@@ -26,22 +26,7 @@ public Appointment(long id, long clientId, long facilityId, long serviceTypeId,
             CreatedBy = createdBy;
         }
 
-        // Edit Appointment
-        public Appointment(long id, long clientId, long facilityId, long serviceTypeId, DateTime appointmentDate, string detail, DateTime? dateCreated, DateTime? dateEdited, long? createdBy, long? editedBy)
-        {
-            AppointmentId = id;
-            ClientId = clientId;
-            FacilityId = facilityId;
-            ServiceTypeId = serviceTypeId;
-            AppointmentDate = appointmentDate;
-            Comment = detail;
-            DateEdited = dateEdited;
-            DateCreated = dateCreated;
-            CreatedBy = createdBy;
-            EditedBy = editedBy;
-        }
-
-        //Add Appointment Attendance
```

Comment:
This too must be maintained, this constructor method is related to Edit Appointment function